adopt-a-dog/styling/stylesheet.css

```diff
@@ -0,0 +1,392 @@
+*{
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+body{
+  background-color: #e6e6e6;
+  width : 100%;
+  max-width: 1450px;
+  min-width: 500px;
+}
+/****** Header ******/
+.page-banner
+{
+  background-image: url("../images/christopher-harris-57366-unsplash.jpg");
+  background-size: 100% 500px;
+  background-repeat: no-repeat;
+  background-position: top;
+  color: white;
+  height: 300px;
+  width: 100%;
+  padding: 10px;
+  display : flex;
+  align-items :center;
+}
+.page-banner-title{
+  padding-left: 30%;
+}
+.page-banner-title h1{
+  color: #4d9900;
+  font-size: 1.5rem;
+  position: absolute;
+  left: 8%;
+  top: 150px;
+  }
+.logo-top{
+ width: 30%;
+ height: 15%;
+ max-width: 35rem;
+ min-width: 15rem;
+ min-height: 10rem;
+ position: absolute;
+ left: 100px;
+ top: 20px;
+}
+.shopping-icon
+{
+ width: 5%;
+ min-width: 0.8rem;
+ height: 1.5rem;
+ position: absolute;
+ top: 30px;
+ right: 180px;
+}
+.banner-price{
```

Comment:
Watch your code formatting. The curly braces and indentation is a little off.

should look like . 

.selector {
  // css indented in 2 spaces (or one tab)
}

Reply:
I  corrected the formatting. I hope readability is better.

Reply:
There are a few places still where it's off still. Give it another run through, the entire css file